docs/source/getting-started-setup.md

```diff
@@ -129,6 +129,11 @@ Follow these steps:
    ./install_requirements.sh
    ```
 
+   **Optional:** Install ExecuTorch as an editable installation:
```

Comment:
so is it a two step process?
i.e. 1. run install_requirements.sh, and 2. do another pip install with these flags?

Reply:
Yes, or you could replace:
`pip install . --no-build-isolation` in install_requirements.sh with:
`pip install --editable . --config-settings editable_mode=strict --no-build-isolation`
and then run install_requirements.sh.

Reply:
yeah if its purely additive step after `install_requirements.sh` then its the cleanest IMHO,